plugins/formal-verification/formver.common/src/org/jetbrains/kotlin/formver/ContextErrorInfo.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver
+
+sealed class ContextErrorInfo {
+    data object Unknown : ContextErrorInfo()
+    data object CondBoolEffect : ContextErrorInfo()
+    data object CondNullEffect : ContextErrorInfo()
+    data object CondNotNullEffect : ContextErrorInfo()
```

Comment:
The effects in this PR are not conditional.

Reply:
Those data objects have been renamed to `ReturnsNullEffect` and `ReturnsNotNullEffect`